src/index.ts

```diff
@@ -29,4 +43,44 @@ const run = async () => {
 	}
 };
 
+/* eslint-disable @typescript-eslint/naming-convention */
+const installCLI = async (): Promise<void> => {
+	const currentFile = url.fileURLToPath(import.meta.url);
+	const currentDir = path.dirname(currentFile);
+	const parentDir = path.resolve(currentDir, "..");
+
+	// Execute bash script
+	const cmdOut = await exec.getExecOutput(
+		`sh -c "` + parentDir + `/entrypoint.sh"`,
+	);
+
+	// Add path to 1Password CLI to $PATH
+	const outArr = cmdOut.stdout.split("\n");
+	if (outArr[0] && process.env.PATH) {
+		const cliPath = outArr[0]?.replace(/^(::debug::OP_INSTALL_DIR: )/, "");
+		core.addPath(cliPath);
+	}
+};
+
+const loadSecrets = async (shouldExportEnv: boolean) => {
+	// Pass User-Agent Inforomation to the 1Password CLI
+	setClientInfo({
+		name: "1Password GitHub Action",
+		id: "GHA",
+		build: semverToInt(version),
+	});
+
+	// Load secrets from environment variables using 1Password CLI.
+	// Iterate over them to find 1Password references, extract the secret values,
+	// and make them available in the next steps either as step outputs or as environment variables.
+	const res = await exec.getExecOutput(`sh -c "op env ls"`);
```

Comment:
Do we need a try catch for this, or any sort of error checking?

Reply:
It is very unlikely for this command to fail. All it does is find the environment variables that have a secret reference as a value. If there are none found, that's also fine. 😄